tests/app_test.py

```diff
@@ -0,0 +1,30 @@
+import requests
+import unittest
+import urllib
+from flask import Flask
+from flask_testing import LiveServerTestCase
+
+
+class BaseTest(LiveServerTestCase):
+    def create_app(self):
+        app = Flask(__name__)
+        app.config["TESTING"] = True
+        app.config["LIVESERVER_PORT"] = 5000
+        app.config["LIVESERVER_TIMEOUT"] = 3
+        return app
+
+    def test_server_is_up_and_running(self):
+        response = urllib.request.urlopen(self.get_server_url())
+        self.assertEqual(response.code, 200)
+
+    def test_example_access_token_check(self):
+        response = requests.get(self.get_server_url() + "/example_access_token_check")
+        self.assertEqual(response.status_code, 403)
```

Comment:
I also encountered an error here on flask end (error very similar to [this](https://stackoverflow.com/questions/46727787/runtimeerror-there-is-no-current-event-loop-in-thread-in-async-apscheduler), line 7 at okta_helper.py: `loop = asyncio.get_event_loop()` )
The error msg is `raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Thread-8'.`
A workaround is changing `loop = asyncio.get_event_loop()` in oktaHelper.py to 
```
loop = asyncio.new_event_loop()
asyncio.set_event_loop(loop)
```
However as the [comment](https://stackoverflow.com/a/46750562) below this workaround, this might be a bad fix to this error. I encountered this error multiple times when I was attempting to add frontend buttons to call /example_access_token api but have no clue why it happens.

Reply:
PM you for more discussion about this issue.